### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by juajum**
- [DEMO LINK](https://juajum.github.io/layout_miami/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,338 @@
...
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>BOSE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
```
Comment (illiarerun) [inline src/index.html:12]: Checklist 1st item

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,338 @@
...
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
```
Comment (illiarerun) [inline src/index.html:14]: Header should stretch for the whole viewport height
image

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,338 @@
...
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="/" class="logo">
```
Comment (illiarerun) [inline src/index.html:18]: Checklsit 5th and 6th items

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,338 @@
...
+            <div class="place__description">
+              Headphones & earbuds
+            </div>
+          </div>
+
+          <div class="place__button-container">
+            <div class="place__button-position">
+              <a href="/" class="button">
+                View all
+              </a>
```
Comment (illiarerun) [inline src/index.html:236]: Checklist 5th item
image

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,338 @@
...
+            </div>
+          </div>
+          <h1 class="header__title">
+            The world shades. Your music shines.
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
```
Comment (illiarerun) [inline src/index.html:42]: It's weird that I click on menu on the right side of the page and it comes out from the left side

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,338 @@
...
+              <a href="#now-on-view" class="menu__link">Now on view</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#testimonials" class="menu__link">Testimonials</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
```
Comment (illiarerun) [inline src/index.html:78]: These all should to some sections on the page, add corresponding ids

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,338 @@
...
+
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">Address</div>
+                <a href="https://goo.gl/maps/Nm8UbU8zDC5M8t479" class="contact-us__info-link">
+                  400 first ave.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
```
Comment (illiarerun) [inline src/index.html:324]: Checklist 9th item

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,338 @@
...
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">Phone</div>
+                <a href="tel:+1 234 5555-55-55" class="contact-us__info-link">
+                  +1 234 5555-55-55
+                </a>
+              </div>
+
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">Phone</div>
+                <a href="email:hello@miami.com" class="contact-us__info-link">
```
Comment (illiarerun) [inline src/index.html:311]: `email:` does nothing, `mailto:` is a way to go

File: src/index.html [hunk 9]
```diff
@@ -1,13 +1,338 @@
...
+                placeholder="Message"
+                class="
+                        contact-us__field
+                        form-field
+                        form-field--textarea"
+              ></textarea>
+              <a href="button" class="button">Send</a>
+            </form>
+
+            <div class="contact-us__info-container">
```
Comment (illiarerun) [inline src/index.html:301]: Add some hover effects here
image

File: src/index.html [hunk 10]
```diff
@@ -1,13 +1,338 @@
...
+                placeholder="Email"
+                class="contact-us__field form-field">
+              <textarea
+                name="message"
+                placeholder="Message"
+                class="
+                        contact-us__field
+                        form-field
+                        form-field--textarea"
+              ></textarea>
```
Comment (illiarerun) [inline src/index.html:297]: Checklist 14th item

File: src/index.html [hunk 11]
```diff
@@ -1,13 +1,338 @@
...
+              <a href="button" class="button">Send</a>
```
Comment (illiarerun) [inline src/index.html:298]: That's what I see when I click on this button
image

File: src/index.html (hunk 5 above)
Comment (illiarerun) [reply src/index.html:42]: Checklist 18th item
image

**Review by illiarerun — CHANGES_REQUESTED** [review]
Nice start, now let's work on fixes:

**Review by Olexandr6 — APPROVED** [review]
Good job🔥

**Pull request closed without merging**